actions.py

```diff
@@ -99,6 +104,10 @@ def push_notification(
                 'is True '
             )
         else:
+            persons = [
+                person.id for person in persons if isinstance(person, Person)
```

Comment:
The else condition will not be handled here, which means that only those ids will be counted for which person instance is passed.